src/loaders/OmeZarrLoader.ts

```diff
@@ -170,29 +167,32 @@ class OMEZarrLoader extends ThreadableVolumeLoader {
 
     // Create one `ZarrSource` per URL
     const sourceProms = urlsArr.map(async (url, i) => {
-      const store = new WrappedStore<RequestInit>(new FetchStore(url), queue);
+      const store = new FetchStore(url);
       const root = zarr.root(store);
 
       const group = await zarr
         .open(root, { kind: "group" })
         .catch(wrapVolumeLoadError(`Failed to open OME-Zarr data at ${url}`, VolumeLoadErrorType.NOT_FOUND));
 
+      // OME-Zarr v0.5+ metadata has a similar shape to v0.4 (for our purposes), but is stored under the `ome` key
+      const meta = (group.attrs as { ome?: unknown }).ome ?? group.attrs;
```

Comment:
I suggest putting this in a function that returns a OMEZarrMetadata as a starting point to completely separating/encapsulating the metadata parsing.